requirements.txt

```diff
@@ -9,7 +10,7 @@ GitPython==3.1.32
 google-cloud-aiplatform==1.35.0
 google-cloud-storage==2.10.0
 Jinja2==3.1.2
-litellm==1.31.10
+litellm==1.34.42
```

Comment:
Initial version to support claude3 on vertex ai is [v1.34.28.dev12](https://github.com/BerriAI/litellm/releases/tag/v1.34.28.dev12).

[v1.34.42](https://github.com/BerriAI/litellm/releases/tag/v1.34.42) is last version of v1.34.x and released on 11th April.
We may update to [v1.37.19](https://github.com/BerriAI/litellm/releases/tag/v1.37.19-stable) without any efforts.
But if we update to [v1.37.20](https://github.com/BerriAI/litellm/releases/tag/v1.37.20), then we need to update openai's version.

```bash
# error message to use litellm>=1.37.20

ERROR: Cannot install pr-agent and pr-agent==0.2.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    pr-agent 0.2.2 depends on openai==1.13.3
    litellm 1.37.20 depends on openai>=1.27.0
```

Reply:
[v1.35.38](https://github.com/BerriAI/litellm/releases/tag/v1.35.38) was worked fine, but [v1.36.0](https://github.com/BerriAI/litellm/releases/tag/v1.36.0) emits error in my env

```bash
# v1.36.0

...
    from openai.types.beta.threads.message_content import MessageContent
ModuleNotFoundError: No module named 'openai.types.beta.threads.message_content'
```